en/docs/developer-guides/workspace-management/overview.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+WSO2 Integrator: BI introduces workspaces – a powerful way to organize and manage complex integration projects through an intuitive graphical interface.
+
+> Note: This feature is available from WSO2 Integrator: BI 1.5.0 onwards and Ballerina version 2201.13.0+.
```

Comment:
Let's use the following instead of`>` to emphasize this more.

```
???+ Note
    This feature...
```
Check https://github.com/wso2/docs-bi/blob/07c98eba094c6f1ebe353d1cd3c0ade728f9a99c/en/docs/deploy/overview.md?plain=1#L21

<img width="1181" height="178" alt="Image" src="https://github.com/user-attachments/assets/9329dd0d-0f9a-4f41-9e31-7583f692d63c" />